docker/docker-compose.yml

```diff
@@ -1,49 +1,50 @@
-version: '2'
+version: '3.3'
 
 services:
 
 ### OMS CORE        #######################################
-### Applications Code Container #############################
-
-    omscore-data:
-        image: tianon/true
-        volumes:
-            - "../:/var/www"
-            - shared:/var/shared
-
-
 ### Workspace Utilities Container ###########################
 
     omscore:
         build:
-          context: ./omscore/
+          context: ./$PATH_OMS_CORE/omscore/
           dockerfile: ./Dockerfile.dev
         image: aegee/omscore:dev
-        volumes_from:
-            - omscore-data
+        volumes:
+            - type: bind
+              source: ./$PATH_OMS_CORE/../
+              target: /var/www
+            - type: volume
+              source: shared
+              target: /var/shared
```

Comment:
how can other volumes refer to this? (Just for curiosity, I still have to read 3.3 docs, I am still at 3)

Reply:
This is a reference, not a declaration :)
The declaration is at the bottom, the volumes section. This tells Docker to mount the named volume 'shared' to `/var/shared` in the VM. (for type: `volume`)
For the type: `bind` I was unable to find a way to reuse it in different containers other than simply restating the same bind path (which really is fine).

Reply:
Oh and btw, it is moved back to `3.2`, Travis did not support `3.3` yet.